subsys/emul/i2c/emul_sbs_gauge.c

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Emulator for SBS 1.1 compliant smart battery fuel gauge.
+ */
+
+#define DT_DRV_COMPAT sbs_sbs_gauge
+
+#define LOG_LEVEL CONFIG_SENSOR_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(sbs_sbs_gauge);
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/emul.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/i2c_emul.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/drivers/spi_emul.h>
```

Comment:
Why are you including SPI headers?

Reply:
Whoops. Missed that, thank you. It was included since this was largely copied from the emul_bmi which does make use of SPI. Thanks!